cabal-install/cabal-install.cabal

```diff
@@ -27,7 +27,35 @@ Extra-Source-Files:
   README.md bash-completion/cabal bootstrap.sh changelog
 
   -- Generated with '../Cabal/misc/gen-extra-source-files.sh | sort'
-  tests/PackageTests/Freeze/my.cabal
+  tests/IntegrationTests/exec/common.sh
+  tests/IntegrationTests/exec/should_run/Foo.hs
+  tests/IntegrationTests/exec/should_run/My.hs
+  tests/IntegrationTests/exec/should_run/adds_sandbox_bin_directory_to_path.sh
```

Comment:
Yes. This is also part of the reason for some of the > /dev/null bits. (I may have forgotten to remove some that weren't strictly necessary.).

The problem is the you get the "configuring blahblah..." message which contains the executable name and if you're trying to compare the contents of stdout with a file...

(There was a little back and forth on this on the original branch. I decided to keep the *option* of having .out and .err files, but in some cases I had to replace them with a "cabal ... | grep ..." to avoid the .exe problem.)